test/system/250-systemd.bats

```diff
@@ -125,4 +125,13 @@ function service_cleanup() {
     service_cleanup
 }
 
+@test "podman generate - systemd - template" {
+    cname=$(random_string)
+    run_podman create --pull=always --name $cname $IMAGE top
+
+    run_podman generate systemd --new --template -n $cname
+    is "$output" ".* ${cname}.%I .*" "%I identifier next to the container name"
+    is "$output" ".*%i.*" "%i indentifiers in template"
```

Comment:
It would be good to add some failure tests.

--template without --new
--template on a pod.

Reply:
DONE :+1: